### DISCUSSION TITLE
PublishOptions in async_nats

### DISCUSSION THREAD

**Discussion by andrenth** [nats-io/nats.rs#710] [General]
Hello

I'm trying to port a publisher from the `nats` crate to `async_nats`, but the `PublishOptions` type doesn't exist in the latter. Is this something that will still be developed or should I look for an alternative way to set these options?

Thanks

**Comment by Jarema** [comment]
Which options do you mean specifically?

**Comment by andrenth** [comment]
In my case it's `expected_last_sequence`.

**Comment by Jarema** [comment]
Ah, sure, will add those.

Will be part of the next release.
Until then, you can insert those headers manually:
```Nats-Expected-Last-Sequence``` key.

**Comment by andrenth** [comment]
Oh that’s great, thanks!

**Comment by Jarema** [comment]
implementation: https://github.com/nats-io/nats.rs/pull/728 

Will land with next release.
